gymfc/envs/assets/gazebo/plugins/msgs/State.proto

```diff
@@ -24,15 +24,20 @@ message State
   optional float  vbat_voltage = 11;
   optional float  vbat_current = 12;
 
+  // Distance Sensor
+  repeated float  distance_sensor = 13 [packed=true];
```

Comment:
lets rename this to `distance`, its just the state we care about not the sensor. others should be refactored in the future.

in the comment please indicate how units are defined and how they are indexed